jdaviz/core/template_mixin.py

```diff
@@ -16,6 +22,12 @@ def __new__(cls, *args, **kwargs):
         obj = super().__new__(cls, *args, **kwargs)
         obj._app = app
 
+        # give the vue templates access to the current config/layout
+        obj.config = app.state.settings.get("configuration", "unknown")
```

Comment:
I think this package ships with a "default" layout, in case you think that is appropriate. There is no "unknown" layout.

p.s. This PR has conflicts now.